identity-apps-core/apps/recovery-portal/src/main/webapp/password-recovery.jsp

```diff
@@ -303,31 +320,62 @@
                         </div>
 
                         <%
-                            if (isEmailNotificationEnabled && isNotificationBasedPasswordRecoveryEnabledByTenant
-                                                    && isQuestionBasedPasswordRecoveryEnabledByTenant ) {
+                            Boolean multipleRecoveryOptionsAvailable = isEmailRecoveryAvailable && isSMSRecoveryAvailable
```

Comment:
Group the related checks

Reply:
Keep one tab for left space

Reply:
Addressed with https://github.com/wso2/identity-apps/commit/92391d27c88da1fa44eb80d1119749f39fbb258d